packages/app/package.json

```diff
@@ -21,7 +21,7 @@
         "@yaireo/tagify": "3.8.0",
         "bootstrap": "^4.5.0",
         "brace": "^0.11.1",
-        "ckeditor4-react": "^4.0.0",
+        "ckeditor4-react": "4.3.0",
```

Comment:
This locks the CKEditor React package to the last version using the underlying CKEditor version 4.22.1 (Standard).